examples/adjoints/adjoints_ex1/adjoints_ex1.C

```diff
@@ -285,6 +285,11 @@ int main (int argc, char ** argv)
   libmesh_example_requires(libMesh::default_solver_package() != INVALID_SOLVER_PACKAGE,
                            "--enable-petsc, --enable-trilinos, or --enable-eigen");
 
+  // This example relies on exceptions to control which Partitioner gets built.
+#ifndef LIBMESH_ENABLE_EXAMPLES
+  libmesh_example_requires(false, "--enable-exceptions");
+#endif
+
```

Comment:
This looks good for now, but in the long run we ought to add a Factory method which tests for a name in the map, then switch adjoints_ex1 to use that rather than attempting a build and catching the error exception.

Reply:
What am I even doing, this should be `#ifndef LIBMESH_ENABLE_EXCEPTIONS`! Will fix right now.

Reply:
We both stared *right* at it...

Reply:
Must be that thing where your brain can still read words even when several of the letters are missing.

Reply:
I'm always tempted to be annoyed with my youngest when she just reads the first letter or two of a word and then guesses wrong at the rest, but damn, at least *she* has an *excuse*...